garage/tf/core/gru.py

```diff
@@ -0,0 +1,62 @@
+"""GRU in TensorFlow."""
+import tensorflow as tf
+from tensorflow.python.ops.gen_array_ops import broadcast_to
+
+
+def gru(name,
+        gru_cell,
+        all_input_var,
+        step_input_var,
+        step_hidden_var,
+        output_nonlinearity_layer,
+        hidden_state_init=tf.zeros_initializer(),
+        hidden_state_init_trainable=False):
+    """
+    Gated Recurrent Unit (GRU).
+
+    Args:
+        name (str): Name of the variable scope.
+        gru_cell (tf.keras.layers.Layer): GRU cell used to generate
+            outputs.
+        all_input_var (tf.Tensor): Place holder for entire time-seried inputs.
+        step_input_var (tf.Tensor): Place holder for step inputs.
+        step_hidden_var (tf.Tensor): Place holder for step hidden state.
+        output_nonlinearity (callable): Activation function for output dense
+            layer. It should return a tf.Tensor. Set it to None to
+            maintain a linear activation.
+        hidden_state_init (callable): Initializer function for the
+            initial hidden state. The functino should return a tf.Tensor.
+        hidden_state_init_trainable (bool): Bool for whether the initial
+            hidden state is trainable.
+
+    Return:
+        outputs (tf.Tensor): Entire time-seried outputs.
```

Comment:
do you mean time-series? please ignore if time-seried is a word.

Reply:
Will fix